templates/_components/_image.twig

```diff
@@ -0,0 +1,177 @@
+{# _components/_image #}
+
+{#
+  Image
+
+  Generate image transforms and render images
+
+  @param   asset        {AssetFileModel}    Image asset to be rendered
+  @param   preset       {string}            Image usage preset
+  @param   class        {string}            Class names to apply to the img tag
+  @param   alt          {string}            Value of the alt attribute for the img tag
+  @param   lazyload     {bool}              Lazyload image or not
+#}
+
+
+
+{# Parameters #}
+{% set imageAsset = params.asset ?? false %}
+{% set imagePreset = params.preset ?? false %}
+{% set imageClass = params.class ?? '' %}
+{% set imageAlt = params.alt ?? false %}
+{% set imageLazyload = params.lazyload ?? true %}
+{% set imageIsCss = not not (params.background ?? false) %}
+{% set imageCssClass = params.background ?? '' %}
+
+
+
+{# Variables #}
+{% set queries = {
+  '--sm-plus':  '(min-width:' ~ 576 ~ 'px)',
+  '--md-plus':  '(min-width:' ~ 768 ~ 'px)',
+  '--lg-plus':  '(min-width:' ~ 992 ~ 'px)',
+  '--xl-plus':  '(min-width:' ~ 1200 ~ 'px)',
+  '--retina':   '(min-resolution: 192dpi)',
+} %}
+{% set presets = {} %}
+{# {% set imageAsset = imageAsset and not craft.embeddedAssets.isEmbedded(imageAsset) ? imageAsset : false %} #}
```

Comment:
We can get rid of this line and any other instances of `craft.embeddedAssets`

Reply:
Removed that line in the next to next commit.